components/InteractionDetailsItem/InteractionDetailsItem.js

```diff
@@ -0,0 +1,18 @@
+import Image from "next/image";
+import {
+  DataContainer,
+  StyledDetailsItem,
+} from "./InteractionDetailsItem.styled";
+
+export default function InteractionDetailsItem({ icon, iconAlt = "", value }) {
+  return (
+    <StyledDetailsItem>
+      <div>
+        <Image src={icon} alt={iconAlt} width={20} height={20} />
+      </div>
```

Comment:
I wondered why you nest the `Image` component in a `div` element One possible explanation for this could be that you're trying to treat the `Image` component like a block-level element, since the `img` element is not a block-level element by default - a clever little trick. 😁

However, it would be cleaner and more flexible to control the styling and layout of the `Image` component directly via CSS. That way you could avoid the redundant div and simplify your code. ✨

Reply:
I delete the surrounding div container at the InteractionDetailsItem.js and ContactDetailsItem.js